tls-client/main.cpp

```diff
@@ -52,10 +52,14 @@ namespace {
 
 const char *HTTPS_SERVER_NAME = "developer.mbed.org";
 const int HTTPS_SERVER_PORT = 443;
-const int RECV_BUFFER_SIZE = 600;
+
+const size_t RECV_BUFFER_SIZE = 600;
+const size_t CRT_BUFFER_SIZE = 1024;
```

Comment:
I think the naming is confusing. This is not a buffer for certificates but for messages about certificates.